clippy_lints/src/utils/conf.rs

```diff
@@ -586,6 +586,26 @@ define_Conf! {
     /// for _ in &mut *rmvec {}
     /// ```
     (enforce_iter_loop_reborrow: bool = false),
+    /// Lint: UNNECESSARY_BLOCKING_OPS.
+    ///
+    /// List of additional blocking function paths to check.
+    ///
+    /// #### Example
+    ///
+    /// ```toml
+    /// blocking-ops = ["my_crate::some_blocking_fn"]
+    /// ```
+    (blocking_ops: Vec<String> = <_>::default()),
+    /// Lint: UNNECESSARY_BLOCKING_OPS.
+    ///
+    /// List of additional blocking function paths to check, with replacement suggestion function paths.
+    ///
+    /// #### Example
+    ///
+    /// ```toml
+    /// blocking-ops-with-suggestions = [["my_crate::some_blocking_fn" , "my_crate::use_this_instead"]]
+    /// ```
+    (blocking_ops_with_suggestions: Vec<[String; 2]> = <_>::default()),
```

Comment:
It would be nicer if these were the same config value. Easiest would be interpreting `["foo::bar", ""]` as having no suggestion. A custom deserializer would let you accept both `"foo::bar"` and `["foo::bar", "baz::bar"]`.

Reply:
I have modified the `clippy_lints::conf::DisallowedPath` enum and re-use it... hope it'll be fine